src/app/portlets/dot-edit-page/content/components/dot-edit-page-view-as-controller/dot-edit-page-view-as-controller.component.ts

```diff
@@ -46,6 +46,10 @@ export class DotEditPageViewAsControllerComponent implements OnInit {
             .subscribe((messages: { [key: string]: string }) => {
                 this.messages = messages;
             });
+
+        this.dotPageStateService.contentAdded$.subscribe(() => {
+            this.pageState.state.haveContent = true;
```

Comment:
Can we have side effect for changing the state of the page like this? Do we need a take one? what happen when we add a content and then remove it?